src/TwigComponent/src/Twig/TwigPreLexer.php

```diff
@@ -39,7 +39,7 @@ public function preLexComponents(string $input): string
             return $input;
         }
 
-        $this->input = $input;
```

Comment:
I am surely wrong but, removing whitespaces won't increase the difficulty for the developper to know where it failed when we indicate the line number in an exception?

Reply:
I think i did this for these tests 

```php
$this->line += substr_count($content, "\n");
```

(and Twig Lexer start by doing this too, soo we need to keep the same line numbers)

Reply:
Sorry ii think i missunderstood at first. 

We are not removing any whitespace here (Twig Lexer will do it), we're just normalizing the new lines to simplify things, but there is no change as `\r\n` and `\r` are new line markers as `\n`